Makefile

```
@@ -185,14 +189,25 @@ choco-package: ## Build a Chocolatey package
 test: fmt ## Run Tests
 	${GO} test `go list ./... | grep -v test/e2e` -timeout 60m -race -coverprofile coverage.txt ${GOTEST_VERBOSE}
 
-.PHONY: e2e-cli-core
-e2e-cli-core: ## Run CLI Core E2E Tests
-	$(eval export PATH=$(ROOT_DIR)/bin:$(ROOT_DIR)/hack/tools/bin:$(PATH))
+.PHONY: e2e-cli-core ## Execute all CLI Core E2E Tests
+e2e-cli-core: e2e-cli-plugin-compatibility-test e2e-cli-tmc-test 
+	${GO} test `go list ./test/e2e/... | grep -v test/e2e/context/tmc | grep -v test/e2e/plugins_compatibility` -timeout 60m -race -coverprofile coverage.txt ${GOTEST_VERBOSE}
+
+.PHONY: e2e-cli-plugin-compatibility-test ## Execute CLI Core Plugin Compatibility E2E test cases
+e2e-cli-plugin-compatibility-test:
+	$(eval export TANZU_CLI_PRE_RELEASE_REPO_IMAGE=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL))
+	@if [ "${TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL}" = "" ]; then \
+		echo "***Skipping Plugin Compatibility test cases because environment variables TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL is not set***" ; \
+	else \
+		${GO} test ./test/e2e/plugins_compatibility -timeout 60m -race -coverprofile coverage.txt ${GOTEST_VERBOSE} ; \
```

```suggestion
		TANZU_CLI_PRE_RELEASE_REPO_IMAGE=$(TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL) ${GO} test ./test/e2e/plugins_compatibility -timeout 60m -race -coverprofile coverage.txt ${GOTEST_VERBOSE} ; \
```

We can update this and remove `L198` where we are exporting the variable.